independent-projects/arc/processor/src/main/java/io/quarkus/arc/processor/BeanInfo.java

```diff
@@ -1265,11 +1279,16 @@ Builder interceptionProxy(InterceptionProxyInfo interceptionProxy) {
             return this;
         }
 
+        Builder checkActive(Consumer<MethodCreator> checkActiveConsumer) {
```

Comment:
Even if it's not public it would not hurt to document the purpose of `checkActiveConsumer`. Here or on the `checkActiveConsumer` field. It's not quite obvious.

Reply:
Sounds fair. I didn't bother, because all the other `Consumer<MethodCreator>` fields/methods are not documented either, but I admit that's a pure excuse for being lazy :-)

Reply:
Well, the purpose of `creatorConsumer` and `destroyerConsumer` is IMO more obvious. This whole "inactive" thing is a bit unusual and special..

Reply:
Added javadoc to `BeanConfiguratorBase.checkActive()` and `BeanInfo.isActive()`. That is the public API we expose here. (`InjectableBean.isActive()` and `checkActive()` were already documented.)

I didn't add javadoc to `BeanInfo.checkActiveConsumer` or `BeanInfo.Builder.checkActive()`, because that is not a public API.

Reply:
> I didn't add javadoc to BeanInfo.checkActiveConsumer or BeanInfo.Builder.checkActive(), because that is not a public API.

I didn't mean the javadoc but merely some comment that explains what it is used for...

Reply:
Added a short comment to the `BeanInfo.checkActiveConsumer` field as well.